### DISCUSSION TITLE
Device Discovery Issue

### DISCUSSION THREAD

**Discussion by bert-mccutchen** [bert-mccutchen/wizrb#7] [Q&A]
@ppostma I'm having an issue where I can't discover my devices. I'm assuming since they are on a 2.4ghz network, while my computer is on a 5ghz network. Though I can connect to them fine if I know their IP.

I used to be able to do this, is this something you can do?

I was having this issue before the v1.1 update so it's not that.

**Comment by ppostma** [comment]
Hi @bert-mccutchen,

I've experimented a bit with the device discovery and noticed that it's not very reliable; most of the time it works for me, but occasionally I only get one response (of the 2 lights I have). After a retry both devices are discovered again.

I tried sending multiple broadcast messages and refactored a bit here: https://github.com/ppostma/wizrb/commit/6f8cbe99fdc740806262982458e06a2b2f9a8636
This seems to improve a bit, but still sometimes I get no reply from one of the devices....

**Comment by ppostma** [comment]
Not sure if the Wifi network is causing it, my computer is also on 5ghz and can still discover devices (well most of the time...)

**Reply by bert-mccutchen** [reply]
Turns out it is 100% my network. My router seems to have released an update that breaks 5Ghz <-> 2.4Ghz communication.

Looks like it's time for me to finally make the switch to a UDM Pro.
